ui/app/AppLayouts/Onboarding2/LoginWithKeycardFlow.qml

```diff
@@ -81,12 +81,11 @@ SQUtils.QObject {
 
             state: {
                 switch (root.authorizationState) {
-                case Onboarding.ProgressState.Success:
+                case Onboarding.AuthorizationState.Authorized:
                     return KeycardEnterPinPage.State.Success
-                case Onboarding.ProgressState.InProgress:
+                case Onboarding.AuthorizationState.InProgress:
                     return KeycardEnterPinPage.State.InProgress
-                // workaround by mapping all failures as wrong pin (#17289)
-                case Onboarding.ProgressState.Failed:
+                case Onboarding.AuthorizationState.WrongPin:
```

Comment:
Now, what about the `Error` state here @micieslak ?

Reply:
Error state is irrelevant here, errors are detected externally, particular pages don't have to care about them